### ISSUE TITLE
Bitbucket to port 443

### ISSUE THREAD

**Issue by TimWolters** [Block8/PHPCI#643]
I've been trying to get phpci to pull a repository from bitbucket but as port 22 is blocked here i wanted to get through the port 443 as being the other option. As i tried to build the project i get the error that the protocol couldn't be handled 

Port 443 [bitbucket](https://confluence.atlassian.com/display/BITBUCKET/Use+the+SSH+protocol+with+Bitbucket#UsetheSSHprotocolwithBitbucket-SSHonPort443)

Error: [error](http://imgur.com/B1z5QWV)

`[2014-11-11 08:39:59] RunCommand.ERROR: ←[0;31mError trying to execute: git clone  -b master git@bitbucket.org:ssh://git@altssh.bitbucket.org:443/mitttig/matas-offertes.git "C:\xampp\htdocs\phpci\PHPCI/build/project2-build28/"←[0m {"buildID":"28"} []`

`[2014-11-11 08:39:59] RunCommand.ERROR: ←[0;31mCloning into 'C:\xampp\htdocs\phpci\PHPCI/build/project2-build28'... fatal: I don't handle protocol 'git@bitbucket.org:ssh' ←[0m {"buildID":"28"} []`

Options: [options](http://imgur.com/EQZTaKJ)

**Comment by niksonleonard** [comment]
Mates, Im with the same problem here.
There no solution for "fatal: I don't handle protocol 'git@bitbucket.org:ssh'"
At this machine I must use only 443 port request.
Do you have some idea to solve it?

**Comment by chayka** [comment]
Hey guys, not sure if that helps, but I spent some time figuring out, so maybe you'll find my tip helpful.

When you create a "github" or "bitbucket" project, in "Repository Name / URL (Remote) or Path (Local)" field you should paste a "short" version of repository name. 
E.g. you have a repository:
"git@github.com:username/project.git"
then you should specify only 
"username/project"
as a field value.

As an alternative you can select "Remote URL", but then you'll probably have to specify "https" repository url: 
https://github.com/username/project.git

And to developers I should say, it is quite not-so-obvious logic. Would it be so hard to parse that username/project part from two forms of URLs (https/git) that people paste? Before I turned to source code, I thought that this PHPCI is completely broken and abandoned.
